accounts-cluster-bench/src/main.rs

```diff
@@ -939,6 +939,9 @@ fn main() {
                 .takes_value(true)
                 .value_name("RPC_BENCH_TYPE(S)")
                 .multiple(true)
+                .requires_ifs(&[
+                    ("token-accounts-by-owner", "mint"),
```

Comment:
It's an array, because we're going to add `token-accounts-by-delegate` in a separate PR.